zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/UserTaskController.java

```diff
@@ -33,6 +42,8 @@ public class UserTaskController {
 
   private final UserTaskServices userTaskServices;
 
+  @Autowired private UserTaskServices userTaskServices;
```

Comment:
❌ This is duplicated now, the services are already provided on the `main` branch.